core/server/data/importer/importers/data/base.js

```diff
@@ -19,6 +19,19 @@ class Base {
             returnDuplicates: true
         };
 
+        this.legacyKeys = {};
+        this.legacyMapper = function legacyMapper(item) {
+            item = _.mapKeys(item, function matchLegacyKey(value, key) {
+                return self.legacyKeys[key] || key;
+            });
+
+            item = _.mapValues(item, function alterDefaultLanguage(value, key) {
+                return (key === 'locale' && value === 'en_US') ? 'en' : value;
```

Comment:
With post and user defaults being null and no way to have changed language defaults for general settings previously I am wondering if we even need to care about this. 

When ghost is initialised the default_locale will be set to 'en' in any case so perhaps we can just omit these attributes to reduce unnecessary warnings as is done with postsPerPage. We can then remove this complexity completely.